luigi/contrib/spark.py

```diff
@@ -189,12 +146,18 @@ def hadoop_conf_dir(self):
         return configuration.get_config().get("spark", "hadoop-conf-dir", None)
 
     def get_environment(self):
-        env = os.environ.copy()
+        env = super(SparkSubmitTask, self).program_environment()
```

Comment:
not a big fan of calling methods on the parent class. should only be doable for constructors